package.json

```diff
@@ -67,6 +67,7 @@
         "fastify": "^4.26.0",
         "fluent-ffmpeg": "^2.1.2",
         "googleapis": "^131.0.0",
+        "i18n-ai-translate": "^1.2.0",
```

Comment:
Do we need this as part of our dependencies if we're only using/installing it during this GH action?

Reply:
If yes, it should be a dev dependency